python/pyspark/pandas/config.py

```diff
@@ -183,6 +183,40 @@ def validate(self, v: Any) -> None:
             "Index type should be one of 'sequence', 'distributed', 'distributed-sequence'.",
         ),
     ),
+    Option(
```

Comment:
Mind regenerating the table? See also above:

```
# NOTE: if you are fixing or adding an option here, make sure you execute `show_options()` and
#     copy & paste the results into show_options
#     'docs/source/user_guide/pandas_on_spark/options.rst' as well.
#     See the examples below:
#     >>> from pyspark.pandas.config import show_options
#     >>> show_options()
```

Reply:
good catch! will update this